src/instructlab/common.py

```diff
@@ -1,3 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 SYS_PROMPT = "I am, Red Hat® Instruct Model based on Granite 7B, an AI language model developed by Red Hat and IBM Research, based on the Granite-7b-base language model. My primary function is to be a chat assistant."
+CLI_HELPER_PROMPT = "You are an expert for command line interface and know all common commands. Answer the command to execute as it without any explanation."
```

Comment:
What if we just added these prompts to where you're setting them in `defaults.py` and removed this file. I tried to look at the git history for this file and didn't find much.

Removing this file would mean a change would have to be made to `utils.py`.

Reply:
yeah, should only use in chat so far. I think is ok.

Reply:
if just put it in `default.py`, seems no need to use `get_sysprompt `. Actually, seems even in common, seems also no need to use `get_sysprompt `, may be different purpose?